internal/rpmmd/test/repository_test.go

```diff
@@ -228,14 +228,14 @@ func TestOldWorkerRepositoryCompatMarshal(t *testing.T) {
 		repo     rpmmd.RepoConfig
 	}{
 		{
-			repoJSON: []byte(`{"Name":"fedora","BaseURLs":["http://example.com/fedora"],"Metalink":"","Mirrorlist":"","GPGKeys":null,"GPGCheck":false,"RepoGPGCheck":false,"IgnoreSSL":false,"MetadataExpire":"","RHSM":false,"ImageTypeTags":null,"PackageSets":null,"Baseurl":"http://example.com/fedora"}`),
+			repoJSON: []byte(`{"Name":"fedora","BaseURLs":["http://example.com/fedora"],"Metalink":"","Mirrorlist":"","GPGKeys":null,"GPGCheck":null,"RepoGPGCheck":null,"IgnoreSSL":false,"MetadataExpire":"","RHSM":false,"ImageTypeTags":null,"PackageSets":null,"Baseurl":"http://example.com/fedora"}`),
```

Comment:
nitpick: seeing the `null` in the JSON string, I think that it would be good to eventually use dedicated structures for marshaling or at least add JSON tags to the `RepoConfig` struct and make these `omitempty`. However not a blocker for this PR from my PoV.

Reply:
Oh I did add JSON tags to RepoConfig, but I did it in a later commit... thinking about this further, I probably should have done that first.

Reply:
Sorry, I was reviewing the PR commit by commit and thinking about this now, you are right that you added tags in later commit, but I didn't get back to my comment. Adding them first would be probably better.

Reply:
Yeah no, I figured you did them commit by commit... I've added the json tags now in the commit you've commented on.

Reply:
Yeah no, I figured you did them commit by commit... I've added the json tags now in the commit you've commented on.